README.md

```diff
@@ -182,11 +180,10 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [Munin](https://github.com/Neo23x0/munin) - Online hash checker for Virustotal and other services.
 * [PowerSponse](https://github.com/swisscom/PowerSponse) - PowerSponse is a PowerShell module focused on targeted containment and remediation during security incident response.
 * [PyaraScanner](https://github.com/nogoodconfig/pyarascanner) - Very simple multithreaded many-rules to many-files YARA scanning Python script for malware zoos and IR.
-* [rastrea2r](https://github.com/aboutsecurity/rastrea2r) - Allows one to scan disks and memory for IOCs using YARA on Windows, Linux and OS X.
+* [rastrea2r](https://github.com/rastrea2r) - Allows one to scan disks and memory for IOCs using YARA on Windows, Linux and OS X.
```

Comment:
why not use `https://github.com/rastrea2r/rastrea2r` link here ?

Reply:
Would indeed be the better version. Thanks